report_artifact_test.go

```diff
@@ -147,7 +147,7 @@ func TestUpload(t *testing.T) {
 			bucketConf: BucketConfiguration{
 				Region: s3Region,
 			},
-			hasErr: true,
+			dryRunNoErr: true,
```

Comment:
You can remove this line, `dryRunNoError` only needs to be set when `hasErr` is set to true

Reply:
Also, to make tests pass change line 144 to `Bucket: s3Name`